src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/ColorMatchingSectionsRequest.cs

```diff
@@ -0,0 +1,48 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.ExecutionPlan.Contracts
+{
+    public class ColorMatchingSectionsParams
+    {
+        /// <summary>
+        /// First query execution plan for comparison.
+        /// </summary>
+        public ExecutionPlanGraphInfo FirstExecutionPlanGraphInfo { get; set; }
+
+        /// <summary>
+        /// Second query execution plan for comparison.
+        /// </summary>
+        public ExecutionPlanGraphInfo SecondExecutionPlanGraphInfo { get; set; }
+
+        /// <summary>
+        /// Flag to indicate if the database name should be ignored
+        /// during comparisons.
+        /// </summary>
+        public bool IgnoreDatabaseName { get; set; }
+    }
+
+    public class ColorMatchingSectionsResult
```

Comment:
Lets rename this class to GraphComparisonResult instead of ColorMatchingSectionResult

Reply:
I assume that means you'll want to update the other names in this file as well (including the file itself)